src/CTA.WebForms2Blazor/FileConverters/ViewFileConverter.cs

```diff
@@ -1,26 +1,93 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
+using System.Text;
 using System.Threading.Tasks;
+using CTA.WebForms2Blazor.ControlConverters;
 using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.WebForms2Blazor.Helpers;
+using HtmlAgilityPack;
 
 namespace CTA.WebForms2Blazor.FileConverters
 {
     public class ViewFileConverter : FileConverter
     {
-        public ViewFileConverter(string sourceProjectPath, string fullPath) : base(sourceProjectPath, fullPath)
+        private Dictionary<String, ControlConverter> ControlRulesMap; //From constructor or service
```

Comment:
Can we make the ControlRulesMap a constant/static value? It can be placed in its own class or in a constants class. If we do this, we can remove it as a parameter from the constructor.

Reply:
I don't think I can make it a constant since it's difficult to create a constant compile time dictionary in C#, but I can make it a static variable in the same class and remove it from the constructor. It just means we will have to update that dictionary manually everytime we add support for a new control type

Reply:
Sure, making it a static variable would work. It's also ok for us to maintain this data structure- it would serve as an indicator as to which controls we're currently supporting. For organizational purposes though, it could be moved to a separate class since it will grow over time. We should also consider making it `readonly` since it shouldn't be modified during runtime.